modules/nw-http2-haproxy.adoc

```diff
@@ -14,6 +14,11 @@ To enable the use of HTTP/2 for the connection from the client to HAProxy, a rou
 
 The connection from HAProxy to the application pod can use HTTP/2 only for re-encrypt routes and not for edge-terminated or insecure routes. This restriction is because HAProxy uses Application-Level Protocol Negotiation (ALPN), which is a TLS extension, to negotiate the use of HTTP/2 with the back-end. The implication is that end-to-end HTTP/2 is possible with passthrough and re-encrypt and not with insecure or edge-terminated routes.
 
+[WARNING]
+====
+WebSocket support over HTTP/2 requires HAProxy 2.4 or later which is unsupported in {product-title} at this time. See the _Networking_ section of the {product-title} release notes for more information on the current supported version of HAProxy.
```

Comment:
This warning must **must** mention re-encrypt routes because websockets do work for edge terminated and insecure routes, even for HTTP/2. I worked through and documented the permutations here: https://github.com/frobware/haproxy-hacks/tree/master/OCPBUGS-173#testing

Reply:
Thanks Andrew, I've updated the warning to mention re-encrypt routes